src/subdomains/supporting/bank-tx/bank-tx/entities/bank-tx.entity.ts

```diff
@@ -219,7 +219,11 @@ export class BankTx extends IEntity {
   //*** GETTER METHODS ***//
 
   get user(): User {
-    return this.buyCrypto?.user ?? this.buyCryptoChargeback?.user ?? this.buyFiat?.user;
+    return (
+      this.buyCrypto?.user ??
+      this.buyCryptoChargeback?.user ??
+      (this.buyFiats.length === 1 ? this.buyFiats?.[0].user : undefined)
+    );
```

Comment:
`this.buyFiats?.[0]?.user`
Or should it return undefined if there are multiple buyFiats?